pyccel/ast/utilities.py

```diff
@@ -189,19 +189,6 @@ def builtin_import(expr):
 
     return []
 
-#==============================================================================
```

Comment:
in what case was this method used before ?

Reply:
In no case that I could find. I have no idea what it was originally written for